fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -358,13 +362,14 @@ def __init__(
         # TODO(anj): Should we conditionally do this only if we have params?
         # TODO(anj): Figure out if we can allocate the buffer during sharding.
         self.buffer_size = sum(p.numel() for p in params)
+        self.ssd_directory = ""
```

Comment:
move this below the boolean check for self.ssd_offload?

Reply:
I can't do that because mypy will complain about ssd_directory being undefined on line 388:
```
 387         self._fsdp_wrapped_module: nn.Module = FlattenParamsWrapper(
 388             module, param_list=to_be_flatten_params, ssd_offload=self.ssd_offload, ssd_directory=self.ssd_directory
 389         )
```